src/app/modules/shared_models/collectibles_entry.nim

```diff
@@ -30,6 +30,8 @@ QtObject:
       generatedId: string
       generatedCollectionId: string
       tokenType: TokenType
+      website: string
```

Comment:
do we really need separate variables? we could use/update the `data` itself

Reply:
sure can do!